notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<p>better to set <code>mag_limit=8</code>,then use that variable in the method call and in the log statement  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
👍